kola/tests/systemd/journald.go

```diff
@@ -110,7 +113,25 @@ storage:
 }
 
 func journalUser(c cluster.TestCluster) {
-	c.MustSSH(c.Machines()[0], "journalctl --user --unit hello.service | grep Foo")
+	if err := util.Retry(10, 2*time.Second, func() error {
+		cmd := "journalctl --user"
+		log, e := c.SSH(c.Machines()[0], cmd)
+		if e != nil {
+			return fmt.Errorf("Did not get expexted log output from '%s': %v", cmd, e)
+		}
```

Comment:
Right, but wouldn't we overwrite the `err` variable created in [line 116](https://github.com/flatcar/mantle/blob/e8d9164b934a8575362a9a8f35d1ba09ed55587c/kola/tests/systemd/journald.go#L116)? So if the retry loop aborts, [line 133](https://github.com/flatcar/mantle/blob/e8d9164b934a8575362a9a8f35d1ba09ed55587c/kola/tests/systemd/journald.go#L133) would print the SSH error instead of whatever we returned from the retry loop?

Reply:
Done - resolving.